medor_delivery_address/report/report_invoice.xml

```diff
@@ -2,24 +2,21 @@
 <odoo>
 	<data>
 		<template id="report_invoice_document_inherit_sale" inherit_id="medor_sexy_invoice.theme_invoice_medor_document">
-	        <xpath expr="//div[@name='invoice_address']" position="attributes">
-	            <attribute name="groups">!sale.group_delivery_invoice_address</attribute>
-	        </xpath>
-	        <xpath expr="//div[@name='invoice_address']" position="before">
+	        <xpath expr="//address[@t-field='o.partner_id']/.." position="replace">
 	            <div class="col-xs-6" groups="sale.group_delivery_invoice_address">
-	                <div t-if="o.partner_shipping_id and (o.partner_shipping_id != o.partner_id)">
+	                <div t-if="o.address_shipping_id and (o.address_shipping_id != o.partner_id)">
 	                    <strong>Shipping address:</strong>
-	                    <div t-field="o.partner_shipping_id"
-	                        t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": True}'/>
+	                    <div t-field="o.address_shipping_id"
+	                        t-field-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": true}'/>
 	                </div>
 	            </div>
-	
+
 	            <div class="col-xs-5 col-xs-offset-1" groups="sale.group_delivery_invoice_address">
 	                <div t-field="o.partner_id"
-	                    t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": True}'/>
-	                <div t-if="o.partner_id.vat" class="mt16"><t t-esc="o.company_id.country_id.vat_label or 'TIN'"/>: <span t-field="o.partner_id.vat"/></div>
+	                    t-field-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": true}'/>

```

Comment:
ok to have a t-field-options if you can't do it in an another way see : https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#qweb

Reply:
I changed to this, to use the same as in `medor_sexy_invoice`. I will try with `t-options`. :)

Reply:
J'ai essayé juste avec `t-options`, mais ça n'affiche rien… Enfin juste le nom du partner. Peut-être qu'il y a quelque chose à changer dans le contenu du champs `t-options`.

J'ai quand même été voir le code, mais en 9.0 je ne trouve pas le bout de code référencé par le lien. À mon avis, le changement sera à faire quand on passera le module en 10.0.